integration/coredns_test.go

```diff
@@ -63,3 +64,37 @@ func (s *CoreDNSSuite) TestCoreDNSVersion140(c *check.C) {
 	s.waitKubectlExecCommand(c, argSlice, "whoami")
 	s.unInstallHelmMaesh(c)
 }
+
+func (s *CoreDNSSuite) TestCoreDNSVersion152(c *check.C) {
```

Comment:
could these be abstracted to a single function `TestCoreDNSVersion(version string, c*check.C)`?
Since the only thing that seems to change is the string in the `setCoreDNSVersion` method?